aiohttp/test_utils.py

```diff
@@ -144,14 +144,14 @@ async def start_server(self, **kwargs: Any) -> None:
     async def _make_runner(self, **kwargs: Any) -> BaseRunner:
         pass
 
-    def make_url(self, path: str) -> URL:
+    def make_url(self, path: Union[str, URL]) -> URL:
```

Comment:
If these changes are necessary, please submit them in a separate PR with a separate changelog fragment.

Reply:
removed